rust/capture/src/v0_request.rs

```diff
@@ -272,6 +272,14 @@ impl RawEvent {
             _ => Ok(distinct_id.chars().take(200).collect()),
         }
     }
+
+    pub fn extract_is_cookieless_mode(&self) -> Result<bool, CaptureError> {
+        match self.properties.get("$cookieless_mode") {
+            Some(Value::Bool(b)) => Ok(*b),
+            Some(_) => Err(CaptureError::InvalidCookielessMode),
+            None => Ok(false),
+        }
+    }
```

Comment:
Ah, interesting, this actually prevents my earlier concern where people could pass a `"true"` string, but this will fail here. Do we expose any kind of information to our customers when we fail to ingest a specific event?

Reply:
Hmm, we do have ingestion warnings in the pipeline but not in capture. I'm not even sure we'd 400 here (though I'll try it).

I'm not too concerned about this - people *really* shouldn't be doing this. I'd care if it could break something else, but if all it means is that their events get dropped, that's fine